src/dpsvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/GetValidOpenDFCMCase.java

```diff
@@ -20,6 +21,13 @@
 public class GetValidOpenDFCMCase {
```

Comment:
Class naming change to GetValidOpenDFCMCaseController?

Reply:
Yes